v3/src/components/case-table/case-table-model.ts

```diff
@@ -11,11 +11,9 @@ export const CaseTableModel = TileContentModel
     type: types.optional(types.literal(kCaseTableTileType), kCaseTableTileType),
     // key is attribute id; value is width
     columnWidths: types.map(types.number),
-  })
-  .volatile(self => ({
     // entire hierarchical table scrolls as a unit horizontally
-    scrollLeft: 0
-  }))
+    horizontalScrollOffset: 0
+  })
```

Comment:
Rather than eliminating the `volatile` property, my inclination would be to follow the pattern we use elsewhere for such ephemeral properties that we actually want to store in the document of manipulating the `volatile` property during the running of the app, and then having a `prepareSnapshot` method in the `CaseTableModel` which copies the `volatile` property into the serialized property, and an `afterCreate` method which copies the serialized property into the `volatile` property. Then you don't need `applyModelChange` because you're only modifying a `volatile` property, and there's not as much MST machinery spun up during scrolling operations. Volatile properties are still observable, so your `reaction` would still work. There may be something I'm missing, however, so let me know if there's a reason to prefer this implementation.